src/Runtime/OMResize.inc

```diff
@@ -1,3 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===---------- OMTensorList.cpp - OMTensor C/C++ Implementation ----------===//
```

Comment:
Typos: `OMTensorList` => `OMResize`. Please change the content of this comment accordingly.